### PR TITLE
WEBSITE-314 Importing in.relation.to into PStore file

### PR DESCRIPTION
**Pull request by hferentschik**
- Adding BZIP2 version of site as posts.pstore.bz2
- Updating importer readme
- Updating test disqus id to otnoitalerni-8
- Disabling spam detection in wxr_exporter, since site is pretty much spam free to avoid false positives
- Adding new ERB files
- Updating Apache redirect config

### CODE REVIEW

**Comment by hferentschik** [comment]
This change can already be viewed on staging (there seems to be an issue w/ comments. I am looking into it)

File: posts/Davide/2015-06-02-hibernate-ogm-42-final-is-out.html.erb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+lace: http://in.relation.to/42822.lace
+
+layout: blog-post
+
+disqus_thread_id: http://in.relation.to/2015/06/02/hibernate-ogm-42-final-is-out
+---
+<div id="documentDisplay" class="documentDisplay">
+
+
+<p class="wikiPara">
```
Comment (gunnarmorling) [inline posts/Davide/2015-06-02-hibernate-ogm-42-final-is-out.html.erb]: Is there a reason for keeping the legacy CSS classes or should we enhance the importer remove them? We don't style them anyways, so they only increase the size of the delivered HTML. Thoughts? Can be done separately of course.

**Comment by gunnarmorling** [comment]
Rebased and applied. Note that I prepended a commit and reworked yours in order to have a stable order of entries in .htaccess_posts. That way the changes to this file in your commit are just limited to the redirects for the added posts.

**Comment by hferentschik** [comment]
> Rebased and applied.

Thanks

> Note that I prepended a commit and reworked yours in order to have a stable order of entries in .htaccess_posts. 

Nice one. I noticed the redirects diff was quite large. Sorting the entries seems to be a good idea.

**Comment by hferentschik** [comment]
> Is there a reason for keeping the legacy CSS classes 

Maybe. I was thinking that at some stage we might want or have to adjust styles depending on whether we are dealing with old posts or new ones. Maybe not, but for now I think it does not hurt to keep them. Also, as you say, it can be done later on.

**Pull request closed without merging**